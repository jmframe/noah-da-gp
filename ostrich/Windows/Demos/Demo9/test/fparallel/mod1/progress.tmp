          10  8.687203171940103E-017  1.300239400521264E-003
