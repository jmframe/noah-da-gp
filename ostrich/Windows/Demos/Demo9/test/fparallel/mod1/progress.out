 solve
     1      0.232E+04      0.128E-02      0.000E+00
     2      0.622E-04      0.131E-02      0.000E+00
     3      0.244E-05      0.131E-02      0.000E+00
     4      0.667E-07      0.130E-02      0.000E+00
     5      0.211E-08      0.132E-02      0.000E+00
     6      0.671E-10      0.128E-02      0.000E+00
     7      0.213E-11      0.131E-02      0.000E+00
     8      0.678E-13      0.129E-02      0.000E+00
     9      0.216E-14      0.128E-02      0.000E+00
    10      0.869E-16      0.130E-02      0.000E+00
 done
 finished
